### ISSUE TITLE
downcase breaks case sensitive repo requests

### ISSUE THREAD

**Issue by blakewhite** [RiotGamesMinions/nexus_cli#99]
We use case sensitive repo names. The following code makes those somewhat problematic to use with this gem.

```
    attribute :repository,
      type: String,
      required: true,
      coerce: lambda { |m|
        m = m.is_a?(String) ? m.gsub(' ', '_').downcase : m
      }
```
and possibly:

```
    # Transforms a given [String] into a sanitized version by
    # replacing spaces with underscores and downcasing.
    # 
    # @param  unsanitized_string [String] the String to sanitize
    # 
    # @return [String] the sanitized String
    def sanitize_for_id(unsanitized_string)
      unsanitized_string.gsub(" ", "_").downcase
    end
```

The result is requests that look like:
```
/nexus/service/local/artifact/maven/resolve?g=com.company.project.product&a=ArtifactName&e=jar&v=1.0.15&r=repo-name
```
which should look like:
```
/nexus/service/local/artifact/maven/resolve?g=com.company.project.product&a=ArtifactName&e=jar&v=1.0.15&r=Repo-Name
```

**Comment by crackmac** [comment]
+1

**Comment by pyro2927** [comment]
Fixed with #108
